### PR TITLE
KafkaSource supports ceOverrides

### LINKED ISSUE #804: Provide a mean on kafka CR event source creation to specify Ce-overrides to be sent with the event.
**Problem**
No means today do Ce-overrides. See knative-eventing pingsoruce as an example

**[Persona:](https://github.com/knative/eventing/blob/master/docs/personas.md)**
me 

**Exit Criteria**
Create a kafkac event source which specifies ce-overrides and then see them in the events received.

**Time Estimate (optional):**
10,000

**Additional context (optional)**
nope

### PR DESCRIPTION
**Pull request by lionelvillard**
Fixes #804

## Proposed Changes

- KafkaSource does not ignore `ceOverrides` anymore
-
-

**Release Note**

```release-note
🎁 KafkaSource now supports ceOverrides.
```

**Docs**

### CODE REVIEW

**Comment by lionelvillard** [comment]
gentle ping.

**Comment by lionelvillard** [comment]
should I reassign to someone else @matzew ?

File: pkg/source/adapter/message.go [hunk 1]
```diff
@@ -161,3 +160,14 @@ func getKeyTypeMapper(keyType string) func([]byte) interface{} {
 	}
 	return keyTypeMapper
 }
+
+type extensionAsTransformer map[string]string
+
+func (c extensionAsTransformer) Transform(r binding.MessageMetadataReader, w binding.MessageMetadataWriter) error {
```
Comment (matzew) [inline pkg/source/adapter/message.go:166]: would this be useful outside?

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**